src/telemetry/telemetry.ts

```diff
@@ -39,7 +40,7 @@ class Telemetry implements ITelemetry {
     this.config = {
       instrumentationKey: this.getInstrumentationKey(),
       disableExceptionTracking: false, // Enables autocollection of uncaught exceptions. Used with `sanitizeStackTrace` telemetry initializer to remove any data that might be PII.
-      disableAjaxTracking: true,
+      disableAjaxTracking: false,
```

Comment:
We will need a filter for Ajax similar to `filterRemoteDependencyData` so that we only capture the data that we need.

Reply:
Something like filterAjaxDependencyData()?

Reply:
As discussed, we will reuse the filterRemoteDependencyData telemetry initializer.